dev/ci/ci-fiat-crypto.sh

```diff
@@ -8,4 +8,4 @@ fiat_crypto_CI_DIR="${CI_BUILD_DIR}/fiat-crypto"
 git_checkout "${fiat_crypto_CI_BRANCH}" "${fiat_crypto_CI_GITURL}" "${fiat_crypto_CI_DIR}"
 ( cd "${fiat_crypto_CI_DIR}" && git submodule update --init --recursive )
 
-( cd "${fiat_crypto_CI_DIR}" && make lite lite-display )
+( cd "${fiat_crypto_CI_DIR}" && make )
```

Comment:
This should be `make coq selected-specific-display`, or, to build even a bit more, `make coq nonautogenerated-specific nonautogenerated-specific-display`.  (The `*-display` targets are not built by default.)

Reply:
Thanks @JasonGross , PR updated. What to build is your call folks.

Reply:
Umm, with this configuration fiat-crypto fails:
```
File "./src/Curves/Weierstrass/Projective.v", line 130, characters 3-7:
Error: Attempt to save an incomplete proof (in proof exceptional_2P_eq_2Q)
```
any idea @JasonGross ?

Reply:
That's with `fiat_crypto_CI_TARGETS='coq nonautogenerated-specific nonautogenerated-specific-display'`

Reply:
https://gitlab.com/coq/coq/-/jobs/68227040

Reply:
Very strange.  It builds fine on master on our travis: https://travis-ci.org/mit-plv/fiat-crypto/jobs/378977060#L895 (This is with 8.master~git~201805141404+19072-0~daily308~ubuntu14.04.1 from https://launchpad.net/~jgross-h/+archive/ubuntu/coq-master-daily)

Reply:
I think this is most likely a bug in Coq on that platform, given that [the line it's pointing at is `Qed`](https://github.com/mit-plv/fiat-crypto/blob/251ea49a661aef7c075ace80867006183ab0cdea/src/Curves/Weierstrass/Projective.v#L130) and the line before that is `par: abstract solve [ fsatz | cbv [id] in * ; fsatz ].` and there should be no way to get that error message after `par: abstract` (I can get `Attempt to save a proof with shelved goals`, but this is a different message.)

Reply:
Interesting, let's rebuild!

Maybe `par` is not fully deterministic, but CI is using flambda on 4.06.0 so maybe that could be a factor.

Reply:
https://gitlab.com/coq/coq/-/jobs/68243830

Reply:
Same error.  I wonder if this is related to the bug that SIGINT in the middle of par crashes/kills coqtop...

Reply:
Yeah something bizarre is going on; I guess that this is due to `par` going OOM in these workers. So what would be a more conservative setting? I'll try with `selected-specific-display`

Reply:
Peak memory usage of 1353868 ko should not make Coq OOM.  (Peak memory usage of the failed file on gitlab is 884004 ko, and peak memory usage of the only file running at the same time is 468468 ko.)  We don't have a target other than `lite` that excludes `Projective.v` (it's included in `coq`).  I could make one (though I'm a bit hesitant to add yet another target to our current dozen or so), but I think it's reasonable to expect that this should work.

Reply:
Maybe you should try running fiat-crypto with `-j1` instead of with `-j2` or whatever?